homeassistant/components/enphase_envoy/quality_scale.yaml

```diff
@@ -0,0 +1,96 @@
+rules:
+  # Bronze
+  action-setup:
+    status: done
+    comment: only actions implemented are platform native ones.
+  appropriate-polling:
+    status: done
+    comment: fixed 1 minute cycle based on Enphase Envoy device characteristics
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
`test_reauth` -> Let's also assert `result` before we start with the `async_configure` part